frontend/src/layout/Sidebar.js

```diff
@@ -323,6 +324,13 @@ function _Sidebar({ user, sidebarCollapsed, setSidebarCollapsed }) {
                         <span className="sidebar-label">Me</span>
                         <Link to={'/me/settings'} onClick={collapseSidebar} />
                     </Menu.Item>
+                    {user.plugin_access?.configure && (
+                        <Menu.Item key="plugins" style={itemStyle} data-attr="menu-item-plugins">
```

Comment:
The plugins were under project settings, but in a quick decision with Paolo we moved them out into their own menu.

The problem is that the plugins UI does two things:
- installs plugins globally
- shows if the plugin is configured globally, but doesn't let you edit it (the logic: we don't want users to see global API keys), though it might be nice as a superadmin to use this interface to install plugins globally.
- configures plugins per project/team

So it's a bit of a frankenscene. Part of it goes under "project settings", part under "instance settings". Thus we "hoisted the scope up" 😅